contrib/sly-mrepl.el

```diff
@@ -1025,13 +1025,15 @@ handle to distinguish the new buffer from the existing."
   (interactive (list (point) t))
   (let* ((point (or point (point)))
          (probe
-          (previous-single-property-change point
-                                           'sly-mrepl--prompt))
+           (previous-single-property-change point
```

Comment:
This seems like an unsolicited indentation change.  Please remove it, it distracts from the purpose of this commit.

Reply:
resolved with a new name binding.